DragaliaAPI/Controllers/DragaliaControllerBase.cs

```diff
@@ -33,6 +33,16 @@ public override OkObjectResult Ok(object? value)
         );
     }
 
+    public OkObjectResult Ok(ResultCode resultCode, object? value)
+    {
+        return base.Ok(
+            new DragaliaResponse<object>(
+                value ?? throw new ArgumentNullException(nameof(value)),
+                resultCode
+            )
+        );
+    }
+
```

Comment:
not sure if this was added by you or just a leftover from rebase, but it is not needed, could you remove it? We have Code() for returning non-1 ResultCodes which are always errors and have no other data with them